src/styles/creators.scss

```diff
@@ -0,0 +1,114 @@
+.creators {
+  margin-bottom: 100px;
+  position: relative;
+
+  @include tablet {
+    display: grid;
+    grid-template-columns: repeat(2, 1fr);
+    grid-gap: 30px;
+    margin-bottom: 130px;
+  }
+
+  @include desktop {
+    max-width: 1280px;
+    margin-bottom: 200px;
+    grid-gap: 120px;
+
+  }
+
+  @include big-desktop {
+    max-width: 1440px;
+  }
+
+  &__description {
+    display: flex;
+    flex-direction: column;
+
+    @include desktop {
+      padding-left: 100px;
+    }
+  }
+
+  &__title {
+    max-width: 280px;
+    position: relative;
+
+    @include tablet {
+      max-width: 100%;
+    }
+
+    @include desktop {
+      max-width: 280px;
+      margin-bottom: 34px;
+    }
+  }
+
+  &__paragraph-last {
+    display: none;
+
+    @include desktop {
+      display: block;
+    }
+  }
+
+  &__paragraphs {
+    display: flex;
+    flex-direction: column;
+    gap: 20px;
+    max-width: 280px;
+    margin-bottom: 30px;
+
+    @include tablet {
+      max-width: 100%;
+      gap: 15px;
+    }
+
+    @include desktop {
+      gap: 30px;
+      max-width: 380px;
+      margin-bottom: 20px;
+    }
+  }
+
+  &__link {
+    margin-bottom: 36px;
+
+    @include desktop {
+      margin-bottom: 50px;
+      width: 86px;
+    }
+  }
+
+  &__button {
+    width: 100%;
+
+    @include tablet {
+      width: 280px;
+    }
+  }
+
+  &__image {
+    @include tablet {
+      display: block;
+      width: 100%;
+      height: 0;
+      padding-bottom: 128.65%;
+      background-image: url(../images/birdie.png);
```

Comment:
The image is hardcoded in your SCSS file. This limits the reusability of your styles. Consider passing it as a variable instead.